docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -1,75 +1,71 @@
 ////
 This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
```

Comment:
?

Reply:
seems that was changed before i rebased :)